GOVERNANCE.md

```diff
@@ -0,0 +1,150 @@
+# Prometheus Governance v1.0
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Prometheus community. Common terminology used in this governance document are listed below:
+
+* **Team members**: Any members of the private [prometheus-team][team] Google group.
+
+* **Maintainers**: Team members who are leads on an individual project ([MAINTAINERS.md][maintainers.md]).
+
+* **Projects**: Any repository in the Prometheus [GitHub organization][gh].
+
+* **The Prometheus Project**: The sum of all activities performed under this governance, concerning one or more Repositories or the community.
+
+## Values
+
+The Prometheus developers and community are expected to follow the values defined in the [CNCF charter][charter], including the [CNCF Code of Conduct][coc]. Furthermore, the Prometheus community strives for kindness, giving feedback effectively and building a welcoming environment. The Prometheus developer community strives to operate by rough consensus for the majority of decisions and only resort to conflict resolution through a motion if consensus cannot be reached.
+
+## Projects
+
+Each project must have a MAINTAINERS.md file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced on the prometheus-users mailing list. Any new projects should be first proposed on the [developers mailing list][devs] following the voting procedures listed below. When a project is no longer relevant, it should be moved to the prometheus-junkyard Github organization.
+
+## Decision making
+
+### Team members 
+
+Team member status may be given to those who have made ongoing contributions for at least 3 months to the Prometheus Project. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account. 
+
+New members may be proposed by any existing member by email to [prometheus-team][team] and voted on by [lazy consensus](#consensus). If no consensus is reached, the new member is not accepted.
```

Comment:
If one member is "I am so convinced that we should *not* have this person in the project that I would rather leave myself than see them join", but their opinion is overridden, who's membership do we value more? I expect that to be the much more common case than wanting a specific new member so badly.

Reply:
@SuperQ @beorn7 it seems like you are the most passionate on this particular point. Could you two get together, exchange your thoughts, and if possible come up with something that works for both of you?

I think calling out "explicit consensus" (with formal announcement and wait time) is a good idea. Where else should it be implemented? Adding new projects/repos?

Reply:
If we vote in a particular candidate X (with super majority), and another member Y leaves because of that, that's a IMHO preferred outcome over first voting out Y and then accepting X by consensus.

To be clear: If we ever are in that situation, we are already deep in the nettles, but a formal governance has to handle those cases, too, or more clearly, it has to handle _precisely_ that case, because we don't even need a formal governance for the happy state of things.

Reply:
@SuperQ could you let us know what your current view is, now that I have tried to explain myself?

Reply:
My proposal was "lazy" consensus with a time limit.  Having no time limit was never part of my idea.

But, I strongly believe in the consensus model, so if @beorn7 objects, we should leave it out.

Really, I don't care much for the governance, I simply wanted to voice my opinion.  Since it doesn't seem like there is anyone in favor of it, I don't care.

Going with super majority voting is fine.

Reply:
Any other objections then? I suggest to explicitly mention that concerns of individuals about a candidate should really be taken seriously (as it should always happen, see the section about the general spirit of consensus) but that the ultimate vote will be the normal super-majority procedure.

Reply:
@beorn7 Sounds good to me (supermajority for accepting new team members).